### DISCUSSION TITLE
greetings

### DISCUSSION THREAD

**Discussion by juliaochoms42** [LesFruitsDefendus/saskatoon-ng#148] [Outreachy]
hello everyone I am called Tayo Julia Nchoum i am super excited to be part of this great team  i am ready to learn know more and also make contributions on this project

**Comment by tristanlatr** [comment]
Hello @juliaochoms42 ,

Nice to meet you.
Thanks for your interest into Saskatoon.
Look forward for your contributions!

Tristan
